src/components/progress-fab/progress-fab.html

```diff
@@ -0,0 +1,13 @@
+<span class="md-button-wrapper">
+    <ng-content></ng-content>
+
+    <div class="md-progress-wrap">
+        <md-progress-circle

```

Comment:
Thanks @marcysutton, I'll try making the process circle a sibling. 

- The role is automatically applied from the Progress Circle Component
- Valuenow, ValueMin, ValueNow are also covered by the circle component. 

Forwarding the `aria-label` / or creating a default one, makes definitely sense. thx